charts/app-config/values-integration.yaml

```
@@ -395,6 +395,12 @@ govukApplications:
               key: bearer_token
         - name: REDIS_URL
           value: redis://shared-redis-govuk.eks.integration.govuk-internal.digital
+        - name: GOOGLE_TAG_MANAGER_ID
+          value: GTM-W573BJPG
+        - name: GOOGLE_TAG_MANAGER_AUTH # Non-prod only
+          value: 2jaYNm4QqCxqsb10EcYo5w
```

Recommend using a YAML anchor for (each of) these values to save repeating them over and over. (Anchors are easy to overuse, but this is probably a good example of where they help.) E.g.
```suggestion
          value: &gtm-id GTM-W573BJPG
        - name: GOOGLE_TAG_MANAGER_AUTH  # Non-prod only.
          value: &gtm-auth 2jaYNm4QqCxqsb10EcYo5w
```

then the rest become

```
        - name: GOOGLE_TAG_MANAGER_ID
          value: *gtm-id
        - name: GOOGLE_TAG_MANAGER_AUTH  # Non-prod only.
          value: *gtm-auth
```
etc.

---

Thanks @sengi 👍🏻 you definitely caught me being verbose there. I've just added YAML aliases and have fixed up the lint errors.